docs/api/client/hoodie.account.rst

```diff
@@ -9,10 +9,399 @@ user </camp/hoodieverse/glossary.html#private-user-store>`__, it makes
 sense to familiarise yourself with **account** before you move on to
 `store </camp/techdocs/api/client/hoodie.store.html>`__.
 
-This is work in progress
-------------------------
+``hoodie-account-client`` is a JavaScript client for the 
+`Account JSON API <http://docs.accountjsonapi.apiary.io/>`_. 
+It persists session information in localStorage (or your own store API) and 
+provides front-end friendly APIs for the authentication-related operations as 
+mentioned above.
 
-Please help us make this awesome <3
+There is also an `admin-specific account client 
+<https://github.com/hoodiehq/hoodie-account-client/blob/master/admin>`_.
 
-For the time being, check out `hoodie-account-client’s
-README <https://github.com/hoodiehq/hoodie-account-client>`__.
+Example
+-------
+
+.. code:: js
+
+    // Account loaded via <script> or require('@hoodie/account-client')
+    var account = new Account('https://example.com/account/api')
+
+    if (account.isSignedIn()) {
+    renderWelcome(account)
+    }
+
+    account.on('signout', redirectToHome)
+
+Constructor
+-----------
+
+.. code:: js
+
+    new Account(options)
+
+table
+
+Returns ``account`` API.
+
+Example
+
+.. code:: js
+
+    new Account({
+        url: '/api',
+        id: 'user123',
+        cacheKey: 'myapp.session',
+        validate: function (options) {
+            if (options.username.length < 3) {
+            throw new Error('Username must have at least 3 characters')
+            }
+        }
+    })
+
+account.ready
+-------------
+
+`Read-only`. Promise that resolves once the account instance loaded its current state from the store.
+
+account.id
+----------
+
+`Read-only`. Returns the account id. Cannot be accessed until the `account.ready <https://github.com/hoodiehq/hoodie-account-client#accountready>`_ promise resolved.
```

Comment:
can we link to `hoodie.account.ready` within this document instead of linking to GitHub?